sdcm/cluster.py

```diff
@@ -3293,7 +3293,7 @@ def _create_session(self, node, keyspace, user, password, compression,
                         # pylint: disable=too-many-arguments, too-many-locals
                         protocol_version, load_balancing_policy=None,
                         port=None, ssl_opts=None, node_ips=None, connect_timeout=None,
-                        verbose=True):
+                        verbose=True, connection_bundle_file=None):
```

Comment:
Did you consider subclassing Cluster for serverless, so we could avoid modifications in favor of rewrite some methods?

Reply:
I did not, since this connection would be also used for scylla-cloud at some point, I don't see how sub-classing would be helpful in this case. 

also we need this support in multiple places, i.e. in multiple stress commands, and also in cqlsh